modules/m4d-implicit-copy-stream/values.yaml

```diff
@@ -35,3 +35,7 @@ copy:
 #        object_key: ""
 #    format: ""
 #    vault: {}
+
+readDataType: "ChangeData"
```

Comment:
Didn't you want to change to LogData?

Reply:
Yes I thought about it and did another PR for the mover. The ChangeData option makes most sense here as a default.
With this configuration we would take Kafka data (which is always a key,value pair) and only take the value part and write this  (e.g. as JSON or parquet) to COS.